angular.json

```diff
@@ -28,7 +28,8 @@
             ],
             "styles": [
               "node_modules/ol/ol.css",
-              "src/scss/styles.scss"
+              "src/scss/styles.scss",
+              "css/all.css"
```

Comment:
The path should read "node_modules/@fortawesome/fontawesome-free/css/all.css". Also, as it is a library you import it should go before "src/scss/styles.scss".